Note that we also unconditionally remove .SUNW_sym.*sort sections since
they point to .symtab section that is removed from a stripped binary.

--- binutils-2.43.1/binutils/objcopy.c
+++ binutils-2.43.1/binutils/objcopy.c
@@ -5008,6 +5008,12 @@ strip_usage (stderr, 1);
 	}
     }
 
+  #if defined(sun) || defined(__sun) || defined(__sun__)
+  handle_remove_section_option(".SUNW_symtabsort");
+  handle_remove_section_option(".SUNW_symtabnsort");
+  handle_remove_section_option(".SUNW_symtlssort");
+  #endif
+
   /* If the user has not expressly chosen to merge/not-merge ELF notes
      then enable the merging unless we are stripping debug or dwo info.  */
   if (! merge_notes_set
